content/collections/experiment-sdks/en/experiment-jvm.md

```diff
@@ -391,6 +391,9 @@ If you're using Amplitude's EU data center, configure the `serverZone` option on
 | `flagConfigPollingIntervalMillis` | The interval to poll for updated flag configs after calling [`Start()`](#start) | `30000` |
 | `flagConfigPollerRequestTimeoutMillis` | The timeout for the request made by the flag config poller | `10000` |
 | `assignmentConfiguration` | Enable automatic assignment tracking for local evaluations. | `null` |
+| `streamUpdates` | Enable streaming to replace polling for receiving flag config updates. Instead of polling every second, Amplitude servers push updates to SDK generally within one second. If the stream fails for any reason, it will revert to polling automatically and retry streaming after some interval. | `false` |
```

Comment:
```suggestion
| `streamUpdates` | Enable streaming to replace polling for receiving flag config updates. Instead of polling every second, Amplitude servers push updates to SDK generally within one second. If the stream fails for any reason, it reverts to polling automatically and retry streaming after some interval. | `false` |
```